src/ngraph/runtime/intelgpu/intelgpu_op_batchnorm.cpp

```diff
@@ -0,0 +1,149 @@
+/*******************************************************************************
+* Copyright 2017-2018 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include <CPP/batch_norm.hpp>
+#include <CPP/concatenation.hpp>
+#include <CPP/scale.hpp>
+#include <CPP/split.hpp>
+
+#include "ngraph/runtime/intelgpu/intelgpu_op_batchnorm.hpp"
+
+#include "ngraph/op/batch_norm.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+// This function converts Shape dimension id into cldnn::concatenation id
+cldnn::concatenation::concatenation_axis get_cldnn_axis(size_t tensor_channel)
```

Comment:
All of the functions in this file should be static so they are not visible outside the file.